vbfml/scripts/err_analysis

```diff
@@ -341,15 +343,19 @@ def average(ctx, input_dir: str, quantity: str, threshold: float) -> None:
         weights = df["weight"][image_mask]
 
         avg_image = np.average(images, axis=0, weights=weights)
-        outdir = pjoin(input_dir, "plots")
+        # outdir = pjoin(input_dir, "plots")
 
         plotter.plot(
             image=avg_image,
             outdir=outdir,
             filename=f"{class_label}_{quantity}_gt_{threshold}.pdf",
+            vmin=1e-3,
+            vmax=6e-2,
             cbar_label="Average Energy (GeV)",
             left_label=process_tag,
-            right_label=class_label,
+            right_label="predict : "
+            + class_label
+            + f"({sum(image_mask)}/{len(image_pixels)})",
```

Comment:
You can use a single f-string here for better code readability, something like this should work:
`f"predict: {class_label} {({sum(image_mask)}/{len(image_pixels)})}"`